src/Libs/GstApp-1.0/GstApp-1.0.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <PackageId>GirCore.GstApp-1.0</PackageId>
+    <RootNamespace>GstApp</RootNamespace>
+    <Description>C# bindings for GStreamer.</Description>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\GObject-2.0\GObject-2.0.csproj" />
```

Comment:
Do you really need GLib and GObject? E.g. GstPbutils works without it: https://github.com/gircore/gir.core/blob/main/src/Libs/GstPbutils-1.0/GstPbutils-1.0.csproj

Reply:
Initalizing GstBase is enough, it initializes GObject, Gst and GLiv, removed